sessionctx/variable/sysvar.go

```diff
@@ -32,6 +32,7 @@ import (
 	"github.com/pingcap/tidb/util/collate"
 	"github.com/pingcap/tidb/util/logutil"
 	"github.com/pingcap/tidb/util/versioninfo"
+	atomic2 "go.uber.org/atomic"
```

Comment:
Why we need a new `atomic` package?

Reply:
Because `FeedbackProbability` in the statistics package is `Float64` from this package, I have to use the same type.